dutree/dutree.py

```diff
@@ -304,10 +305,38 @@ class DuScan:
     "Disk Usage Tree scanner"
 
     def __init__(self, pathname):
+        self._dev_allow = []
+        self._dev_deny = []
+        self._dev = lstat(pathname).st_dev
         self._path = self._normpath(pathname)
         self._tree = None
         self._check_path()
 
+    def skip_other_filesystems(self):
+        "add self._dev to self._dev_allow"
+        self._dev_allow = [self._dev]
+
+    def skip_proc_sys_filesystems(self):
+        "add proc and sys devices to self._dev_deny"
```

Comment:
Here it's more a short summary, so it's slightly more helpful.

But I would go with:
```
"""
Skip /proc and /sys when scanning. There's nothing of interest for us there

Those pseudofiles will mostly be 0-sized, and if they're not, they
represent memory anyway, not something we can clean up.
"""
```

Reply:
removed it